Products/Zuul/routers/template.py

```diff
@@ -290,6 +291,15 @@ def getDataPointDetails(self, uid):
         data =  Zuul.marshal(dict(record=details, form=form))
         return data
 
+    def _getPasswordFields(self, obj):
+        """
+        Return fields for obj if it has xtype and it is password.
+        """
+        fields = IFormBuilder(IInfo(obj)).fields()
+        return [field for field in fields
+                        if fields[field].get('xtype', None)
+                        and fields[field].get('xtype') == 'password']                      
```

Comment:
I'm not familiar with this part of the code. Just wondering if the check should cover a wider range of names.
Is  it possible to have fields with names like `Password` or `passwd`? Or maybe if the string contains `password`.